<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Study with Bunny - Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #E1F5FE; /* M√†u n·ªÅn xanh pastel */</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFF9C4;
     border-radius: 30px;
     padding: 10px;
     color: #000000; /* M√†u ch·ªØ h·ªìng ƒë√†o */
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); /* Hi·ªáu ·ª©ng b√≥ng ch·ªØ */</string>
      </property>
      <property name="text">
       <string>Ch√†o m·ª´ng ƒë·∫øn v·ªõi Study with Bunny üê∞</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_progress">
        <property name="font">
         <font>
          <family>Chalkboard SE</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFDCDC;
     border-radius: 30px;
     padding: 10px;
     color: #000000; /* M√†u ch·ªØ h·ªìng ƒë√†o */
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); /* Hi·ªáu ·ª©ng b√≥ng ch·ªØ */</string>
        </property>
        <property name="text">
         <string>Xem ti·∫øn ƒë·ªô</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_quiz">
        <property name="font">
         <font>
          <family>Chalkboard SE</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFDCDC;
     border-radius: 30px;
     padding: 10px;
     color: #000000; /* M√†u ch·ªØ h·ªìng ƒë√†o */
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); /* Hi·ªáu ·ª©ng b√≥ng ch·ªØ */</string>
        </property>
        <property name="text">
         <string>Quiz vui</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_logout">
        <property name="font">
         <font>
          <family>Chalkboard SE</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFDCDC;
     border-radius: 30px;
     padding: 10px;
     color: #000000; /* M√†u ch·ªØ h·ªìng ƒë√†o */
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); /* Hi·ªáu ·ª©ng b√≥ng ch·ªØ */</string>
        </property>
        <property name="text">
         <string>ƒêƒÉng xu·∫•t ‚Üí</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QLabel" name="bunnyImageLabel">
        <property name="styleSheet">
         <string notr="true">border: 3px solid #F88379; /* Vi·ªÅn h·ªìng ƒë√†o */
</string>
        </property>
        <property name="pixmap">
         <pixmap>image/Logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_start_lesson">
        <property name="font">
         <font>
          <family>Chalkboard SE</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFDCDC;
     border-radius: 35px;
     padding: 10px;
     color: #000000; /* M√†u ch·ªØ h·ªìng ƒë√†o */
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); /* Hi·ªáu ·ª©ng b√≥ng ch·ªØ */</string>
        </property>
        <property name="text">
         <string>B·∫Øt ƒë·∫ßu h·ªçc</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="progressLabel">
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>27</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000; 
     padding: 5px;</string>
      </property>
      <property name="text">
       <string>H√¥m nay b·∫°n ƒë√£ ho√†n th√†nh 0/5 b√†i h·ªçc! C·ªë l√™n nh√© üê∞</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
